src/nbl/video/IOpenGL_LogicalDevice.h

```diff
@@ -761,11 +761,20 @@ class IOpenGL_LogicalDevice : public ILogicalDevice, protected impl::IOpenGL_Log
                 }
             }
 
+            auto raster = params.rasterization;
+            if (gl.isGLES() && !gl.getFeatures()->isFeatureAvailable(COpenGLFeatureMap::NBL_EXT_clip_control))
+            {
+                if (raster.faceCullingMode == asset::EFCM_BACK_BIT)
+                    raster.faceCullingMode = asset::EFCM_FRONT_BIT;
+                else if (raster.faceCullingMode == asset::EFCM_FRONT_BIT)
+                    raster.faceCullingMode = asset::EFCM_BACK_BIT;
+            }
```

Comment:
@Crisspl review this, we've had some issues with Vulkan with @achalpandeyy  as well ;)

Reply:
yeah that's my code, with knowledge I have it's all right
could you point me where clip_control ext is not specific enough?

Reply:
>     4)  Should the face culling behavior be modified in GL_UPPER_LEFT
        clip origin mode?
        RESOLVED:  Yes.
        See how the modifications to section 14.6.1 "Basic Polygon
        Rasterization" negate the sign of the polygon area when the clip
        control origin is GL_UPPER_LEFT.
        Since culling behavior is specified as CW (clockwise) or CCW
        (counter-clockwise) different triangle faces would be culled
        when the clip origin is changed, which would be an unacceptable
        side effect.

Reply:
@achalpandeyy can explain

Reply:
wrapped it up in #276